pegasus/sites.v3/code.org/views/video_with_fallback.haml

```diff
@@ -1,7 +1,7 @@
 - download_path ||= nil
 - poster_path ||= "//i.ytimg.com/vi/#{video_code}/mqdefault.jpg"
 
-%figure{style: "width:100%; height: 100%; margin: 0 auto;"}
+%figure{style: "width:100%; margin: 0 auto;"}
```

Comment:
Does this change cause any visual differences? Just want to know if this will affect other pages that `video_with_fallback` is used

Reply:
This view is used all over the site so styling this specifically in the `tools-styles.scss` sheet is a safer option.

Reply:
yeah I checked a bunch of places where it's used and it seemed fine, but admittedly I probably didn't find every reference. I'll try again to override it, but I was having trouble due to inline style having the highest specificity. I really don't want to use !important if I can help it 😬

Reply:
oh! I'll just move those inline styles to the tools-styles.scss stylesheet

Reply:
Sounds perfect!